src/pages/TeamOverview/TeamOverview.tsx

```diff
@@ -35,42 +36,71 @@ const getTeamLeadCard = (teamLead: UserDataType): JSX.Element => {
     return <Card id={id} data-testid='team-lead' columns={columns} url={`/user/${id}`} navigationProps={teamLead} />;
 };
 
-interface PageState {
-    teamLead?: UserDataType;
-    teamMembers?: UserDataType[];
-}
-
 const TeamOverview = (): JSX.Element => {
     const location: Location<TeamsType> = useLocation();
     const {teamId} = useParams();
-    const [pageData, setPageData] = React.useState<PageState>({});
```

Comment:
Ok, about the `PageState` structure changes: To be able to follow the structure of the `Search` component props, I decided it was better to use an array with all members, keeping the `teamLead` in the **first** position always